README.md

```diff
@@ -36,6 +36,10 @@ or as an environment variable.
 | --sequence-number   |                      | Sequence # to use for board template, needs to be unique     | `99999` |
 | --version           |                      | Display version information                                  | `false` |
 
+**Note:** you can ommit --sequence-number and use default value 99999. You will be 
+prompted to update this when adding template in hound. Automatic template validation
+will suggest next value to be used.
+
```

Comment:
We will have instructions in hound of how to add a new template. Part of the instructions will be adding the template to the existing unit test that will validate sequence number/id and suggest the next value to use. At some point we might have way too many templates to easily count by hand. 😄 🤞